gapic/templates/tests/unit/gapic/%name_%version/%sub/test_%service.py.j2

```diff
@@ -472,7 +472,7 @@ def test_{{ method.name|snake_case }}(transport: str = 'grpc', request_type={{ m
         {% if method.client_streaming %}
         response = client.{{ method.name|snake_case }}(iter(requests))
         {% else %}
-        response = client.{{ method.name|snake_case }}(request)
+        response = client.{{ method_name }}(request)
```

Comment:
Can you please clarify, why `snake_case` is not needed anymore (here and in many other places)?

Reply:
Of course. We define `method_name` near the beginning of the file like so:
```
{% with method_name = method.name|snake_case + "_primitive" if method.operation_service else method.name|snake_case %}
```

Note the underscore: this is `method_name`, not `method.name`.

Reply:
Oh, thanks! (I did not notice  `method_name` vs `method.name` difference and that what made me thing that snake_case just got removed